addons/point_of_sale/views/pos_session_view.xml

```diff
@@ -120,7 +120,7 @@
                         </div>
                     </group>
                     <separator string="Summary by Payment Methods" attrs="{'invisible' : [('state', '=', 'opening_control')]}"/>
-                    <field name="statement_ids" attrs="{'invisible' : [('state', '=', 'opening_control')]}">
+                    <field name="statement_ids" attrs="{'invisible' : [('state', '=', 'opening_control')]}" context="{'dont_validate': True}">
```

Comment:
here you put context {'form_view_ref': 'point_of_sale.view_bank_statement_pos_session'} that context will make it use thi specific view, instead of the one existing in the accounting.